client/python/CONTRIBUTING.md

```diff
@@ -0,0 +1,48 @@
+How to develop on this python client project 
+=
+<hr />
+
+Armada Project welcomes contributions from the opensource community.
+### Note that you need PYTHON 3+ to get this project working and this project does not work on native windows OS.
+
+## Setting up your own fork of this repository on linux, Mac, and WSL OS.
+<hr />
+
+1) On github interface click on the `Fork` button.
+2) Clone your forked of armada repository on your personal git repository by running git clone `github.com:Your_Git_Username/armada.git`.
+3) Enter the directory `cd armada`
+4) Add upstream repository [Armada Repo](https://github.com/G-Research/armada.git)
+
+## Install Pyenv/Python and Poetry
+<hr />
+
+2) Install [Pyenv](https://github.com/pyenv/pyenv) by following the instructions [here](https://realpython.com/intro-to-pyenv/).
+3) Install [poetry](https://python-poetry.org/) on your local machine by following the instructions on [python poetry documentation](https://python-poetry.org/docs/).
+
+## Setting up your own environment
+<hr />
+
+1) Run `make python` then `cd client/python` and run `poetry install` to pull all necessary dependencies.
+
+## Run test to ensure everything is working
```

Comment:
We should encourage people to run tests via tox.

`tox`, from the root of client/python, should be all people need to run to get unit tests and formatting run.